docs/manifest/resources.md

```diff
@@ -49,6 +49,14 @@ The following additional sizes are also supported, but not required.
 - 48x48
 - 64x64
 
+The following image/file extensions are supported:
+- bmp
+- gif
+- exif
+- jpg
+- tiff
+- png
+
```

Comment:
In the Taos example, I see a lot of .svg icons. The browser certainly supports it. Hopefully the native clients can support it too.